cmd/job-trigger-controller-manager/main.go

```diff
@@ -0,0 +1,113 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"os"
+
+	"github.com/bombsimon/logrusr"
+	"github.com/sirupsen/logrus"
+
+	utilerrors "k8s.io/apimachinery/pkg/util/errors"
+	prowv1 "k8s.io/test-infra/prow/apis/prowjobs/v1"
+	"k8s.io/test-infra/prow/flagutil"
+	"k8s.io/test-infra/prow/logrusutil"
+	controllerruntime "sigs.k8s.io/controller-runtime"
+	ctrlruntimelog "sigs.k8s.io/controller-runtime/pkg/log"
+
+	prpqv1 "github.com/openshift/ci-tools/pkg/api/pullrequestpayloadqualification/v1"
+	"github.com/openshift/ci-tools/pkg/controller/prpqr_reconciler"
+)
+
+const (
+	appCIContextName = "app.ci"
+)
+
+type options struct {
+	kubernetesOptions flagutil.KubernetesOptions
+
+	namespace string
+	dryRun    bool
+}
+
+func gatherOptions() (*options, error) {
+	o := &options{kubernetesOptions: flagutil.KubernetesOptions{NOInClusterConfigDefault: true}}
```

Comment:
Is there any reason why we shouldn't use in-cluster config? This controller will only interact with a single cluster (the one where it is running, so I'd guess we could even support in-cluster config exclusively (like the pools-cm does https://github.com/openshift/ci-tools/blob/c3f8a3cdfc64de1120d0c56b8badd690e8c9a6eb/cmd/dptp-pools-cm/main.go#L83-L86)